DAL/LoginDAO.cs

```diff
@@ -10,9 +12,14 @@ namespace DAL
 {
     public class LoginDAO : DAO
     {
-        public void Test()
+        public Employee GetUserByID(string username)
         {
-            List<Ticket> tickets = GetTicketCollection().Find(ticket => ticket.Status == ETicketStatus.Open).ToList();
+            Employee newEmployee = (Employee)GetEmployeeCollection().Find(employee => employee.FirstName == username).FirstOrDefault();
```

Comment:
The cast is not needed because to GetEmployeeCollection is already typed

Reply:
Fixed!